File: Benchmark-normal.py
#!/usr/bin/python
import time
import numpy as np
import sys

def procedure():
    print(np.random.normal(0,1,1))

N = int( sys.argv[1] );   
t1 = time.time()
t0 = time.clock()
for i in range(0,N):
    procedure()
print time.clock() - t0, "seconds process time"
#procedure()
print time.time() - t1, "seconds wall time"

File: Benchmark-normal.pl
#!/usr/bin/perl
use strict;  use Time::HiRes qw( time );  use Math::Random;
my $start = time(); 
my $N = $ARGV[0] || 10;
for (my $i=0; $i < $N; $i++){
  print (random_normal(), "\n");
}
my $end = time();
printf("\n");
printf("%.12f\n", $end - $start);
